front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,347 @@
+                const prevStep = stdcmData.simulation.base.stops[index - 1];
+                return (
+                  <TR
+                    key={index}
+                    style={step.duration !== 0 && !isLastStep ? styles.blueRow : styles.tbody}
+                  >
+                    <TD style={styles.indexColumn}>{index + 1}</TD>
+                    <View style={styles.opWidth}>
+                      <TD style={styles.td}>
+                        {!isFirstStep && !isLastStep && step.name === prevStep.name
+                          ? '='
+                          : step.name || 'Unknown'}
+                      </TD>
+                    </View>
+                    <View style={styles.chWidth}>
+                      <TD style={styles.chColumn}>{step.ch}</TD>
+                    </View>
+                    <View style={styles.trackWidth}>
+                      <TD style={styles.td}>{step.track_name}</TD>
+                    </View>
+                    <View style={styles.endWidth}>
+                      <TD style={styles.stopColumn}>
+                        {isLastStep || step.duration !== 0 ? getStopTime(step.time) : ''}
+                      </TD>
+                    </View>
+                    <View style={styles.passageWidth}>
+                      <TD
+                        style={{
+                          ...(step.duration !== 0 && !isLastStep
+                            ? {
+                                width: `${step.duration < 600 && step.duration > 60 ? 60 : 70}px`,
+                              }
+                            : {}),
+                          ...(step.duration !== 0 && !isLastStep
+                            ? styles.blueStop
+                            : styles.stopColumn),
+                        }}
+                      >
+                        {
+                          // eslint-disable-next-line no-nested-ternary
+                          !isFirstStep && !isLastStep
+                            ? step.duration !== 0
+                              ? getStopDurationTime(step.duration)
+                              : getStopTime(step.time)
+                            : ''
+                        }
+                      </TD>
+                    </View>
+                    <View style={styles.startWidth}>
+                      <TD style={styles.stopColumn}>
+                        {isFirstStep || (step.duration !== 0 && !isLastStep)
+                          ? getStopTime(step.time + step.duration)
+                          : ''}
+                      </TD>
+                    </View>
+                    <View style={styles.weightWidth}>
+                      <TD style={styles.td}>
+                        {!isFirstStep
+                          ? '='
+                          : rollingStockData.mass &&
+                            `${Math.floor(rollingStockData.mass / 1000)} t`}
```

Comment:
We know that rollingStockData.mass is defined :)
```suggestion
                          : `${Math.floor(rollingStockData.mass / 1000)} t`}
```

Reply:
done